landing/templates/landing/learn-more.html

```diff
@@ -4,19 +4,19 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/jpeg" href="{% static '/Pro4U_logo.jpeg' %}" />
-	<link rel="stylesheet" href="{% static 'style.css' %}" />
+    <link rel="icon" type="image/jpeg" href="{% static '/img/Pro4U_logo.jpg' %}" />
+	<link rel="stylesheet" href="{% static '/css/learn-more/style.css' %}" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Pro4U Website Landing Page</title>
   </head>
   <body>
 	<section>
 		<header>
 			<h2>
-				<a href="#" class="logo"><img src="{% static '/Pro4U_logo.jpeg' %}" alt="Facebook link" height="200" width="209" /></img></a>
+				<a href="{% url 'learn_more' %}" class="logo"><img src="{% static '/img/Pro4U_logo.jpg' %}" alt="Facebook link" height="200" width="209" /></img></a>
 			</h2>
 			<div class="navigation">
```

Comment:
Can we move the navigation part to a more generic page? Take a look at [Template inheritance](https://docs.djangoproject.com/en/4.2/ref/templates/language/#template-inheritance)

Reply:
> Can we move the navigation part to a more generic page? Take a look at [Template inheritance](https://docs.djangoproject.com/en/4.2/ref/templates/language/#template-inheritance)

Resolved